revised.tex

```diff
@@ -0,0 +1,555 @@
+Other authors have empirically confirmed the importance of kind and polite social environments
+in open source projects \cite{singh2012,steinmacher2013,steinmacher2018a}.
+Therefore, projects should not just say that they welcome new members:
+they should make a proactive effort to foster positive feelings in them.
+
+\begin{itemize}
+
+\item
+  Post a welcome message on the project's social media pages, Slack channels, forums, or email lists.
+  Projects might consider maintaining a dedicated ``Welcome'' channel or list,
+  where a project lead or community manager writes a short post asking newcomers to introduce themselves.
+
+\item
+  Offer assistance in finding ways to make an initial contribution.
+
+\item
+  Direct the newcomer to project members who have a similar background or skill set
+  so as to demonstrate fit to the newcomer.
+
+\item
+  Point the newcomer to essential project resources (e.g., the contribution guidelines).
+
+\item
+  Clearly identify work items they can start with.
+  A growing number of projects explicitly tag bugs or issues as ``suitable for newcomers'',
+  and ask established members not to fix them
+  in order to ensure there are suitable places for new arrivals to start work.
+  
+\end{itemize}
+
+Projects can further designate one or two members to serve as a point-of-contact for each newcomer.
+Doing this may reduce the newcomer's hesitancy to ask questions,
+particularly when they are told from the outset that there are no dumb questions in the community.
+
+\rulemajor{Rule 2: Help potential contributors evaluate if the project is a good fit.}
+
+People could contribute to many different projects;
+the first and most important step in being welcoming is to help them determine whether
+your project is a good fit for their interests and abilities.
+Their decision to contribute can be related to reputation or external needs,
+but also to a desire to learn or give back to the community.
+In all of these cases,
+the more you help newcomers understand if this is the right project for them,
+the more quickly they will either start contributing or look elsewhere.
+
+To do this,
+the project should explicitly state what the different types of skills required are.
+This information should be easily accessible and guide new members to the tasks they may handle.
+LibreOffice,
+for example,
+provides a way for developers to filter available tasks by required skills and difficulty~\cite{libreoffice-filtered}.
+
+The project should also help developers evaluate their skills,
+since ``basic Python skills'' means very different things to different people.
+Tools like My GitHub Resume~\cite{my-github-resume} and Visual Resume~\cite{sarma2016}
+that aggregate information from previous contributions can help with this assessment,
+while the OpenHatch project~\cite{openhatch} aggregated entry-level issues from a variety of open source projects
+and classified them according to language and other required skills
```

Comment:
```suggestion
and classifies them according to language and other required skills
```

Reply:
Matching tense to the first part of the first part of the sentence.

Reply:
See above.